Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml

```diff
@@ -172,6 +177,10 @@
                         Name="chkEnableAutoUpdates">
                         <Label Content="{Binding Path=(l10n:DisplayString.sSettings_General_UpdateCheck)}" />
                     </CheckBox>
+                    <CheckBox Click="ChkRunAtLogOn_Click"
```

Comment:
Private methods in Cairo use a lower case first letter. Please follow this convention and revert the capitalizations